src/beautifiers/verilog-mode/index.coffee

```diff
@@ -0,0 +1,63 @@
+###
+Requires http://golang.org/cmd/gofmt/
+###
+
+"use strict"
+Beautifier = require('../beautifier')
+path = require("path")
+
+module.exports = class EmacsVerilogMode extends Beautifier
+  name: "Emacs Verilog Mode"
+  link: "https://www.veripool.org/projects/verilog-mode/"
+  isPreInstalled: false
+  executables: [
+    {
+      name: "Emacs"
+      cmd: "emacs"
+      homepage: "https://www.gnu.org/software/emacs/"
+      installation: "https://www.gnu.org/software/emacs/"
+      version: {
+        parse: (text) -> text.match(/Emacs (\d+\.\d+\.\d+)/)[1]
+      }
+    }
+  ]
+
+  options: {
+    Verilog: true
```

Comment:
Using `true` means "all options are supported".

You could change this to the following:

```
options: {
  Verilog: {
    verilog_emacs_script_path: true
  }
}
```

Which means only `verilog_emacs_script_path` option is supported.

Reply:
While some of the other beautifiers use `true` I would like to try and narrow this down in order to prevent future confusing in the automatically generated docs.